flex-config/ui_attributes.common.json

```diff
@@ -20,25 +20,25 @@
         "enabled": true
       },
       "conversation_transfer": {
-        "enabled": false,
+        "enabled": true,
         "cold_transfer": true,
         "multi_participant": true
       },
       "chat_to_video_escalation": {
-        "enabled": false
+        "enabled": true
       },
       "conference": {
         "enabled": true,
         "hold_workaround": false
       },
       "enhanced_crm_container": {
-        "enabled": false,
+        "enabled": true,
         "url": "https://www.bing.com/search?q={{task.from}}",
         "should_display_url_when_no_tasks": false,
         "display_url_when_no_tasks": "https://www.bing.com"
       },
```

Comment:
I think chat-to-video and enhanced-crm should be disabled by default for the following reasons:
- chat-to-video's web app is intended as an example; I don't believe it was considered production ready
- crm container has undesirable results by default and requires configuration of the CRM URL. We could enable this by default if we were to ship an example page to embed via serverless assets, but I don't think we should enable Bing by default.

Reply:
i disagree, we are shipping a production ready solution, we are shipping a development tool - i think we should be show casing the features that are there and let them turn them off if they are not desired.